src/ui/ThreadReplies/index.scss

```diff
@@ -88,4 +88,6 @@
 .sendbird-ui-thread-replies__icon {
   position: relative;
   display: inline-flex;
+  /*rtl:raw:
+  transform: rotate(180deg);*/
```

Comment:
**as-is**
![Screenshot 2024-06-24 at 5 38 55 PM](https://github.com/sendbird/sendbird-uikit-react/assets/10060731/babea03a-d52a-4273-9a8c-9e8f1dff4edf)

**to-be**
<img width="212" alt="Screenshot 2024-06-24 at 5 38 24 PM" src="https://github.com/sendbird/sendbird-uikit-react/assets/10060731/dec379d5-964a-4658-af92-775024bb465b">